Flows/HomeFlow/Sources/HomeFlow/PageView/PageView.swift

```diff
@@ -128,6 +130,30 @@ struct PageView: View {
                 .environmentObject(store.filterStore)
         }
     }
+    
+    func loadingView(shimmerIsActive: Bool) -> some View {
+        GeometryReader { geometry in
```

Comment:
Unfortunately, GeometryReader makes big impact on the redrawing here because the view will be redrawing every time when you are scrolling. Let's try to use `UIScreen.main`.